packages/connect-thegraph/src/queries/index.ts

```diff
@@ -1,8 +1,8 @@
 import gql from 'graphql-tag'
 import * as fragments from './fragments'
 
```

Comment:
Queries are no longer constants, but instead they are functions that return a gql tag. This is because they can now be used with `query` or `subscription`. The type is not being enforced, so connectors should be careful about using the correct strings. I thought that enforcing it would be overkill, but it can be added if you like.

Reply:
Nice, I agree